config/all-in-one/templates/aimanager.yaml

```diff
@@ -40,10 +40,6 @@ spec:
             - name: spec.aiManager.size
               value: '{{ .Values.cp4waiops.profile }}'
             {{- end }}
-            {{- if .Values.cp4waiops.aiManager.imageCatalog }}
```

Comment:
This may need to be changed to `.Values.cp4waiops.imageCatalog` so that the same imageCatalog setting is shared by cp-shared, aimanager, eventmanager.

Reply:
Same with above, `imageCatalog` is not used  in aimanager any more.